clang/docs/RealtimeSanitizer.rst

```diff
@@ -83,3 +83,53 @@ non-zero exit code.
     #13 0x00010230dd64 in main main.cpp:9
     #14 0x0001958960dc  (<unknown module>)
     #15 0x2f557ffffffffffc  (<unknown module>)
+
+Disabling
+---------
+
+In some circumstances, you may want to suppress RealtimeSanitizer violations in a specific scope.
```

Comment:
Minor nit: it's not technically the violations that are suppressed, it's the error messages and action invoked on detecting a violation. I appreciate this is a bit "if a tree falls in a forest, does it make a sound?", but I reckon it's worth clarifying. My suggestion would be to re-word this (and the one on line 92) like "you may want to suppress RealtimeSanitizer errors in a specific scope"